src/pages/my_page/MyPagePage.tsx

```diff
@@ -116,6 +117,26 @@ const MyPagePage = () => {
     onError: () => dispatch(setNotification({ message: t('feedback.error.get_messages'), variant: 'error' })),
   });
 
+  const dialogueNotificationsParams = getDialogueNotificationsParams(user?.nvaUsername);
+
+  const isOnDialoguePage = location.pathname === UrlPathTemplate.MyPageMyMessages;
+  const notificationsQuery = useQuery({
+    enabled: isOnDialoguePage && !!user?.isCreator && !!dialogueNotificationsParams.owner,
+    queryKey: ['notifications', dialogueNotificationsParams],
+    queryFn: () => fetchCustomerTickets(dialogueNotificationsParams),
+    meta: { errorMessage: false },
+  });
+
+  const unreadDoiCount = notificationsQuery.data?.aggregations?.type?.find(
+    (bucket) => bucket.key === 'DoiRequest'
```

Comment:
Kunne disse vært typesterk?
```suggestion
    (bucket) => bucket.key === 'DoiRequest' as TicketType
```

Reply:
Ah, flotters!